onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -115,6 +115,7 @@ def optimize_for_ort(
     config_name: str | None = None,
     *,
     debug: bool = False,
+    clear_metadata: bool = False,
```

Comment:
@titaiwangms : can we change the calls to specify this flag to be true, when desired? Hopefully we have only two or three places where we call this.

We could make the default-value True, but it seems to me for devs/experimenters False is better. But that's an option too

Reply:
Defaulting to false sounds preferable to me